internal/runner/service.go

```diff
@@ -86,6 +91,8 @@ func (r *runnerService) CreateSession(ctx context.Context, req *runnerv1.CreateS
 
 	envs := make([]string, len(req.Envs))
 	copy(envs, req.Envs)
+	envs = append(envs, fmt.Sprintf("RUNME_RUNNER_ADDRESS=%s", r.address))
+	envs = append(envs, fmt.Sprintf("RUNME_RUNNER_TLS_DIR=%s", r.tlsDir))
```

Comment:
It allows us to infer the server params to perform the runme env store snapshot

![image](https://github.com/stateful/runme/assets/43882/214aa0c2-1124-46a3-bccb-3f662706e171)